presto-tests/src/test/java/com/facebook/presto/execution/TestQueues.java

```diff
@@ -374,6 +376,42 @@ public void testQueuedQueryInteraction()
         assertEquals(queryInfo.getErrorCode(), ADMINISTRATIVELY_PREEMPTED.toErrorCode());
     }
 
+    @Test(timeOut = 240_000)
+    public void testEagerPlanValidation()
+            throws Exception
+    {
+        AtomicBoolean triggerValidationFailure = new AtomicBoolean();
+
+        queryRunner.installPlugin(new ResourceGroupManagerPlugin());
+        queryRunner.installPlugin(new TestingPlanCheckerProviderPlugin(triggerValidationFailure));
+        queryRunner.getPlanCheckerProviderManager().loadPlanCheckerProviders();
```

Comment:
I'm not sure if there is a better way to call `loadPlanCheckerProviders()`. In `PrestoServer` it is called during `run()`, but for `TestingPrestoServer` there isn't really an equivalent. It could just be loaded as needed though..

Reply:
this seems fine